rest/src/test/groovy/whelk/rest/api/CrudSpec.groovy

```diff
@@ -399,7 +399,7 @@ class CrudSpec extends Specification {
         when:
         crud.doGet(request, response)
         then:
-        response.getStatus() == HttpServletResponse.SC_NOT_FOUND
+        response.getStatus() == HttpServletResponse.SC_NOT_ACCEPTABLE
     }
 
     def "GET /<id>/data.rdf should return 404 Not Found"() {
```

Comment:
Does this media type (application/rdf+xml) still resolve with 404?

Reply:
I changed the test to match the behavior.
It seemed reasonable since the view /data.rdf does not exist
'Accept: application/rdf+xml' on the resource or data view returns 406 Not Acceptable. (this has later been changed in https://github.com/libris/librisxl/pull/364 where the Accept header is ignored if not acceptable and jsonld served)

The test for data.ttl is broken here. It was fixed on the lxl-2608 branch (https://github.com/libris/librisxl/pull/364)

Reply:
OK, sounds fair.